lib/error.js

```diff
@@ -7,8 +7,6 @@ var messages = require('joi/lib/language').errors
 
 var ERRNO = {
   ACCOUNT_EXISTS: 101,
-  ACCOUNT_LOCKED: 121,
```

Comment:
I don't think we've ever removed one before - comment it out with a note so we don't re-use the errno seems like a good approach to copy.